apps/beeswax/src/beeswax/server/dbms.py

```diff
@@ -338,8 +338,8 @@ def get_query_server_config_via_connector(connector):
       # For connectors/computes, the auth details are not available
       # from configs and needs patching before submitting requests
       'principal': 'TODO',
-      'auth_username': None,
-      'auth_password': '',
+      'auth_username': compute['options'].get('auth_username'),
+      'auth_password': compute['options'].get('auth_password', 'hue'),
```

Comment:
From where is this default `hue` value coming from?

Reply:
this is our current default from our configs. Won't really get used in most cases are we are impersonating and sending the `auth_username` only.